packages/bpk-component-slider/src/form-test.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import BpkSlider from './BpkSlider';
+
+window.ResizeObserver =
+  window.ResizeObserver ||
+  jest.fn().mockImplementation(() => ({
+    disconnect: jest.fn(),
+    observe: jest.fn(),
+    unobserve: jest.fn(),
+  }));
+
+describe('BpkSlider form test', () => {
+  it('should work as a form component in a form', async () => {
+    const Wrap = () => {
+      const [sliderValue, setSliderValue] = useState<number[]>([50]);
+      return (
+        <form data-testid="form">
+          <BpkSlider
+            id="slider"
+            min={0}
+            max={100}
+            value={sliderValue}
+            name="slider"
+            ariaLabel={['slider']}
+            ariaValuetext={['min', 'max']}
+            data-testid="mySlider"
+            onChange={(value) => setSliderValue(Array.isArray(value) ? value : [value])}
```

Comment:
Why is the value sometimes within an array and sometimes not?

Reply:
Because sometimes there aren't a min and max value that could both change and only one side of the slider is changeable. So I wanted one of the tests to check for when both `min, max` can change to cover various types. Does that make sense?

Reply:
<img width="833" alt="image" src="https://github.com/user-attachments/assets/cde88735-841d-4c01-a830-50881d224403">

Reply:
I think I see, so if the `sliderValue={[20,80]}` then the bottom "two thumb" slider will be shown? 

I think that if we want to test both cases (single slider & multi slider) we should create two test cases. Or we choose the most simple case only to test the form interaction.

Reply:
> so if the sliderValue={[20,80]} then the bottom "two thumb" slider will be shown?
yes that is right!
And okay, I can make it be clear in the description too so there is no confusion for users. :)